src/index.ts

```diff
@@ -358,7 +358,8 @@ function createServicePrincipal(azureWebsiteName: string,
     const graphOptions = options;
     graphOptions.tokenAudience = 'graph';
     const baseUri = options.environment ? options.environment.activeDirectoryGraphResourceId : undefined;
-    const graphClient = new GraphRbacManagementClient(new DeviceTokenCredentials(graphOptions), options.domain ? options.domain : '', baseUri);
+    const tokenCredentials = new DeviceTokenCredentials(graphOptions);
+    const graphClient = new GraphRbacManagementClient.GraphRbacManagementClient(tokenCredentials, options.domain ? options.domain : '', baseUri);
```

Comment:
Why did this change? `new GraphRbacManagementClient.GraphRbacManagementClient`

Reply:
I believe it was due to the upgrade of the library versions. After updating I was getting the following error:
`error TS2351: Cannot use 'new' with an expression whose type lacks a call or construct signature.`

According to the [TypeScript docs](https://www.typescriptlang.org/docs/handbook/modules.html), the example for `import m = require("mod")` you must use `m.something`